wayang-api/wayang-api-sql/src/test/resources/model.json

```diff
@@ -1,24 +1,16 @@
 {
-  "version": '1.0',
-  "defaultSchema": 'wayang',
+  "version": "1.0",
+  "defaultSchema": "wayang",
   "schemas": [
     {
-      "name": 'postgres',
-      "type": 'custom',
-      "factory": 'org.apache.wayang.api.sql.calcite.jdbc.JdbcSchema$Factory',
+      "name": "postgres",
+      "type": "custom",
+      "factory": "org.apache.wayang.api.sql.calcite.jdbc.JdbcSchema$Factory",
       "operand": {
-        "jdbcDriver": 'org.postgresql.Driver',
-        "jdbcUrl": 'jdbc:postgresql://localhost:5432/imdb',
-        "jdbcUser": 'postgres',
-        "jdbcPassword": 'postgres'
-      }
-    },
-    {
-      "name": 'fs',
-      "type": 'custom',
-      "factory": 'org.apache.calcite.adapter.file.FileSchemaFactory',
-      "operand": {
-        "directory": 'C:/incubator-Wayang-CrossPlatform/incubator-wayang-SQL/wayang-api/wayang-api-sql/src/test/resources'
+      "jdbcDriver": "org.postgresql.Driver",
+      "jdbcUrl": "jdbc:postgresql://localhost:5432/dvdrental",
+      "jdbcUser": "aditya",
```

Comment:
@AdityaGoel11 Do not delete the fs schema, it is meant to support SQL queries over csv files.